syz-verifier/main.go

```diff
@@ -189,5 +190,11 @@ func main() {
 
 	vrf.startInstances()
 
+	monitor := MakeMonitor()
+	monitor.SetStatsTracking(vrf.stats)
+	monitor.InitHTTPHandlers()
+	log.Logf(0, "run the Monitor at http://127.0.0.1:8080/")
+	go monitor.ListenAndServe("127.0.0.1:8080")
```

Comment:
If there is any error (the port is already occupied (this is a popular one) or any other error), it will be silently ignored, http server won't work and user will be confused.
It's better to handle the returned error.
Also I would let user configure the address, port 8080 is popular, or I may want to run it on 80 to satisfy firewalls, or I may want to listen on external addresses so that it's accessible remotely.

Reply:
Agree.
Didn't done that because of 2 reasons:
1. We use config files. Want to move the configuration there and don't use command line for the configuration.
2. I don't understand why we use multiple configuration files. Want to merge them and move this port there later.

Add TODO?

Reply:
> I don't understand why we use multiple configuration files.

The intention was that user passes existing manager configs as inputs.
Maybe it's not too important. We don't have enough experience in usage of syz-verifier. To date all syz-verifier configs were prepared specifically for syz-verifier.

Reply:
Got it. We have to understand the future integration with syzbot in advance then...